spec/lib/formatter_spec.rb

```diff
@@ -107,9 +107,13 @@ module CodeClimate::TestReporter
     end
 
     describe '#short_filename' do
+      def call(*args)
+        formatter.send(:short_filename, *args)
+      end
+
       it 'should return the filename of the file relative to the SimpleCov root' do
-        expect(formatter.short_filename('file1')).to eq('file1')
-        expect(formatter.short_filename("#{::SimpleCov.root}/file1")).to eq('file1')
+        expect(call('file1')).to eq('file1')
+        expect(call("#{::SimpleCov.root}/file1")).to eq('file1')
```

Comment:
Are these spec changes related to the changes in this commit? They don't seem to be.